### ISSUE TITLE
Crash upon right-clicking

### ISSUE THREAD

**Issue by physkets** [naxuroqa/Venom#466]
#### Describe the bug
The program exits when I right-click inside the application window.

#### To Reproduce
Just right click.

#### Expected behavior
It should not crash?

#### Actual behavior
It crashes with the following terminal output:
```
(venom:29105): Gtk-CRITICAL **: 17:13:15.028: gtk_list_box_row_get_index: assertion 'GTK_IS_LIST_BOX_ROW (row)' failed
**
ERROR:arraylist.c:1197:gee_array_list_real_get: assertion failed: (index >= 0)
Aborted (core dumped)
```

#### Desktop (please complete the following information):
 - OS: Archlinux
 - Version: 0.5.2

#### Additional context
I installed it by compiling it using the build-script available on the AUR:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=venom

**Comment by naxuroqa** [comment]
Thank you for reporting, should be fixed in the next release.

**Comment by naxuroqa** [comment]
Fixed in `0.5.3`
